specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/apimapis.json

```diff
@@ -4415,6 +4415,283 @@
         },
         "x-ms-odata": "./definitions.json#/definitions/TagResourceContract"
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}/wikis/default": {
```

Comment:
>"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}/wikis/default" [](http://example.com/codeflow?start=4&length=151)

You should also define a collection Get API for this:

Get /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}/wikis
which returns an array with one item.

See: [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#get-apis](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#get-apis).